README.md

```diff
@@ -42,6 +42,9 @@ jobs:
       # https://aquasecurity.github.io/trivy/v0.29.2/docs/vulnerability/examples/filter/#by-vulnerability-ids
       # Default: ""
       trivy-ignore-files: .trivyignore
+      # Generate a Software Bill Of Materials (SBOM) report the app in json format. 
```

Comment:
report **for** the app in **JSON** format.

Reply:
@joberget

Reply:
@stigok fixed!